core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -943,7 +952,7 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeOptionalString(routing);
             out.writeOptionalString(parent);
             out.writeLong(version);
-            
+
             out.writeByte(versionType.getValue());
             out.writeLong(autoGeneratedIdTimestamp);
             out.writeVLong(seqNo);
```

Comment:
not related to this change but a (big) bug - this can unassigned.  I wonder how our BWC didn't catch it... maybe we don't run the nodes with assertion enabled?